apps/web/composables/useModernImage/useModernImage.ts

```diff
@@ -21,9 +21,10 @@ export const useModernImage: UseModernImageReturn = () => {
   const addModernImageExtension = (url: string | undefined): string => {
     if (!url) return '';
 
-    const matches = url.match(/(\.\w+)(?:$|\?)/);
+    const matches = url.match(/.?(\.\w+)(?:$|\?)/);
     if (!matches) return url;
-    const baseExtension = String(matches[1]);
+
+    const baseExtension = String(matches[1].split('.').pop());
```

Comment:
without this step, it will always compare for example .jpg against jpg and never add the webp or avif extension